src/task1/remove duplicated image/remove_duplicated.py

```diff
@@ -0,0 +1,84 @@
+import sys
+import json
+import math
+import copy
+import cv2
+
+#set threshold of deleting the gps location
+
+
+#setting command lines. h is height of the image, w is width of the image
+input_json_file=sys.argv[1]
```

Comment:
I recommend switching to a argument parser. It helps a lot with dealing with bad input and makes your programs more usable in general. Look into argparse. If you want an example, look at the bottom of (https://github.com/UTAT-AeRo/vision_201819/blob/master/src/task2/flattening_module/imageflattener.py).